transcribe.py

```diff
@@ -149,6 +149,60 @@ def transcript_file_name_from_video_file_name(video_file_name):
 
 	return transcript_job_name + ".json"
 
+def format_time(val):
+	secs = float(val)
+	hours = int(secs/3600)
+	secs = secs - (hours * 3600)
+	mins = int(secs/60)
+	secs = secs - (mins * 60)
+	m_secs = int(secs*1000)
+	secs = int(secs)
+	return pad_time(hours) + ':' + pad_time(mins) + ':' + pad_time(secs) + ',' + pad_time(m_secs, 3)
+
+def pad_time(val, l= 2):
+	return ('0'.join([''] * (l+1)) + str(val))[-l:]
+
+def convert_transcribe_to_srt(video_file):
+	if not transcript_exists_in_s3(video_file):
+		logger.info("Transcript for video file %s does not exist" % video_file)
+		return "Failed to convert to srt. Transcript does not exist."
+
+	logger.info("Conversion to srt started for video file: %s" % video_file)
+	with open(transcript_file_name_from_video_file_name(video_file), encoding='utf-8') as f:
+		raw = json.load(f)
+		items = raw['results']['items']
+		start = end = counter = index = 0
+		current = float(items[0]['start_time'])
+		output = next_line = ''
+
+		for token in items:
+			start = format_time(current)
+			if token['type'] == 'punctuation':
+				next_line = next_line[0:-1] + token['alternatives'][0]['content']
+				end = format_time(items[counter - 1]['end_time'])
```

Comment:
If the punctuation is at the beginning of the transcript, won't this throw an index-out-of-bounds exception ?